src/components/comment/comment.ts

```diff
@@ -0,0 +1,41 @@
+export const initComments = () => {
+  const commentsWithRepliesEnabled = document.querySelectorAll(
+    '.js-comment-with-replies-enabled'
+  );
+
+  for (const comment of [...commentsWithRepliesEnabled]) {
```

Comment:
No need to spread here! For loops work on iterators, not just arrays (just like how spread works on iterators, not just arrays)

Reply:
Oh gotcha! Is it just `forEach` that you need to spread for?

Reply:
Well you can only use .forEach on objects that have a forEach method on their prototype, (not all iterators, but most). Arrays, NodeLists, Sets, and Maps all have forEach methods, but that is because they each implement that, not because they are all iterators.

Maybe you are thinking of `.map` or other Array methods, which most other iterators don't have implemented.

Reply:
Got it. I think I was getting tripped up because [IE11 doesn't support `nodeList.forEach`](https://caniuse.com/?search=nodelist%20foreach) but that shouldn't be a concern here :+1: